rest-api/src/test/java/gov/cms/qpp/conversion/api/controllers/v2/QrdaControllerV2Test.java

```diff
@@ -0,0 +1,197 @@
+package gov.cms.qpp.conversion.api.controllers.v2;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.isNull;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.http.ResponseEntity;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+import gov.cms.qpp.conversion.ConversionReport;
+import gov.cms.qpp.conversion.Source;
+import gov.cms.qpp.conversion.api.exceptions.AuditException;
+import gov.cms.qpp.conversion.api.model.ConvertResponse;
+import gov.cms.qpp.conversion.api.model.Metadata;
+import gov.cms.qpp.conversion.api.services.AuditService;
+import gov.cms.qpp.conversion.api.services.QrdaService;
+import gov.cms.qpp.conversion.api.services.ValidationService;
+import gov.cms.qpp.conversion.encode.JsonWrapper;
+import gov.cms.qpp.conversion.model.error.TransformException;
+import gov.cms.qpp.test.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class QrdaControllerV2Test {
+
+	private static final String GOOD_FILE_CONTENT = "Good file";
+
+	static final Path validationJsonFilePath = Paths.get("src/test/resources/testCpcPlusValidationFile.json");
+
+	private MultipartFile multipartFile;
+
+	private InputStream validationInputStream;
+
+	@InjectMocks
+	private QrdaControllerV2 objectUnderTest;
```

Comment:
This is a new annotation for me. I might incorporate it into my JsonWrapper story. Nice.